cmake/RunLint.cmake

```diff
@@ -1,18 +1,27 @@
-get_filename_component(LINT_DIR ${LINT_DIR} ABSOLUTE)
-get_filename_component(LINT_PREFIX ${LINT_DIR} PATH)
+get_filename_component(LINT_DIR "${LINT_DIR}" ABSOLUTE)
+get_filename_component(LINT_PREFIX "${LINT_DIR}" PATH)
```

Comment:
If I recall, cmake handles quoting automatically. So the quotes are not needed, and may actually be less reliable.